src/clojure_commons/template.clj

```diff
@@ -0,0 +1,7 @@
+(ns clojure-commons.template
+  (:require [clojure.string :as string]))
+
+(defn render
+  "Takes a format string and a map of interpolation values to substitute into the string."
+  [fmt m]
+  (string/replace fmt #"\{\{([^\}]+)\}\}" (fn [[orig k]] (get m (keyword k) orig))))
```

Comment:
I think this may not work right if there should be literal `{{}}` in the format string, but I doubt that'll come up, so seems fine to me. Just wanted to note it.

Reply:
Ah, that's something good to consider. I'll check it.

Reply:
It looks like it works by default because the regular expression doesn't match `{{}}`. Hopefully I understand the edge case correctly. Let me know if I've missed anything.